contracts/TokenHandler.sol

```diff
@@ -38,8 +38,13 @@ contract TokenHandler is ITokenHandler, ITokenManagerType, ReentrancyGuard, Crea
         /// @dev Track the flow amount being received via the message
         ITokenManager(tokenManager).addFlowIn(amount);
```

Comment:
```suggestion
        _migrateToken(tokenManager, tokenAddress, tokenManagerType);

        /// @dev Track the flow amount being received via the message
        ITokenManager(tokenManager).addFlowIn(amount);
```
call migrate separately so it's cleaner, gas cost difference should be very small

Reply:
you mean call it for every token type? indiscriminately? This would revert if the token in question does not implement `isMinter`